testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -15523,6 +15523,58 @@ void checkBitAnd(SqlOperatorFixture f0, FunctionAlias functionAlias) {
     f0.forEachLibrary(list(functionAlias.libraries), consumer);
   }
 
+  @Test public void testBitCountFunc() {
```

Comment:
add javadoc. explain that one tests BITCOUNT and the other tests BIT_COUNT.

are these methods identical? do you intend them to stay identical? If so, share code, don't copy-paste. We have done that for other operators.

Reply:
They are not identical. `BIT_COUNT()` supports decimal values. I viewed this as a unique feature in MySQL. If it makes more sense, I can use the MySQL behaviour for both `BITCOUNT()` and `BIT_COUNT()`.